### PR TITLE
add proto and direnv docs for dev setup

### PR DESCRIPTION
**Pull request by garrensmith**
A small docs fix to add two things that I needed to do to get the prisma-engine running.

### CODE REVIEW

**Comment by garrensmith** [comment]
I'm going to close this since I've done something wrong with the install.

File: README.md [hunk 1]
```diff
@@ -176,6 +178,16 @@ Other variables may or may not be useful.
 
 Run `cargo test` in the repository root.
 
+## Parallel rust-analzyer builds
```
Comment (tomhoule) [inline README.md:181]: typo: rust-analyzer

**Review by tomhoule — APPROVED** [review]
:100:

File: README.md (hunk 1 above)
Comment (garrensmith) [reply README.md]: Thanks. I've pushed a fix.

**Review by dpetrick — APPROVED** [review]

**Pull request merged**